agent/fw_drupal8.c

```diff
@@ -534,19 +534,29 @@ NR_PHP_WRAPPER(nr_drupal8_name_the_wt_via_symfony) {
 NR_PHP_WRAPPER_END
 
 void nr_drupal_version() {
-  char* string = "Drupal::VERSION;";
-  zval retval;
-  int result
-      = zend_eval_string(string, &retval, "Retrieve Drupal Version");
-  
+  zval* zval_version = NULL;
+  zend_class_entry* class_entry = NULL;
```

Comment:
Maybe add a comment explaining that all this function does is read `Drupal::VERSION` constant in a safe way?

Reply:
Good idea - done with [b3bc447](https://github.com/newrelic/newrelic-php-agent/pull/833/commits/b3bc4476f8f69b924da3b874dc7b5eacfb39a95b) .